pennylane/data/data_manager/__init__.py

```diff
@@ -185,7 +185,9 @@ def _download_datasets(  # pylint: disable=too-many-arguments
 
     if pbar is not None:
         if attributes is None:
-            file_sizes = [int(head(url).headers["Content-Length"]) for url in dataset_urls]
+            file_sizes = [
+                int(head(url, timeout=5).headers["Content-Length"]) for url in dataset_urls
```

Comment:
Just curious about the choice of `5`. We may want to double check this is the datasets channel. Maybe we could add a `timeout` kwarg to this infrastructure to make it customiable?